hri_safety_sense/src/VscProcess.h

```diff
@@ -18,10 +18,8 @@
 /**
  * ROS Includes
  */
-#include "ros/ros.h"
```

Comment:
As with the other header files, rename to `.hpp`.

Reply:
7de7162